moveit_ros/moveit_servo/doc/servo_tutorial.md

```diff
@@ -69,15 +69,15 @@ The bare minimum requirements for running `moveit_servo` on your robot include:
 Because the kinematics are handled by the core parts of `moveit`, it is recommended that you have a valid `moveit_config` package for your robot and you can run the demo launch file included with it.
 
 #### A note on controllers
-As of now, there is no [ros2_controls controller](https://github.com/ros-controls/ros2_controllers) that can work with `moveit_servo`. ROS1 controllers that worked include `position_controllers/JointGroupPositionControllers` and `velocity_controllers/JointGroupVelocityControllers`. 
+As of now, there is no [ros2_controls controller](https://github.com/ros-controls/ros2_controllers) that can work with `moveit_servo`. ROS1 controllers that worked include `position_controllers/JointGroupPositionControllers` and `velocity_controllers/JointGroupVelocityControllers`.
```

Comment:
This is actually outdated now. `position_controllers/JointGroupPositionControllers` and `velocity_controllers/JointGroupVelocityControllers` are available. Suggest simply deleting this line.